Pipfile

```diff
@@ -10,6 +10,7 @@ pytest-mock = "*"
 mock = "*"
 elasticmock = "*"
 v = {editable = true,version = "*"}
+invoke = "*"
```

Comment:
Fjern. invoke er ikke n avhengighet for applikasjonen

Reply:
d633aa1 :)